scripts/builder_setup.sh

```diff
@@ -26,7 +26,7 @@ export CMAKE_VERSION=3.18.2.2
 export GIMME_VERSION=1.5.4
 
 export GO_VERSION=1.17.11
-export IBM_MQ_VERSION=9.2.2.0
+export IBM_MQ_VERSION=9.2.4.0
```

Comment:
The link also needs to be changed, as the file name was changed from `<version>-IBM-MQ-Toolkit-MacX64.pkg` to `<version>-IBM-MQ-DevToolkit-MacX64.pkg`.

Reply:
Thanks, I would have never caught that! Fixed